biophysics-and-physicobiology.csl

```diff
@@ -91,25 +91,25 @@
   <macro name="volume">
     <choose>
       <if type="article-journal" match="any">
-        <text variable="volume" suffix=","/>
+        <text variable="volume" form="short" suffix=","/>
       </if>
       <else>
         <group delimiter=" ">
-          <label variable="volume" form="short"/>
-          <text variable="volume"/>
+          <label variable="number-of-volumes" form="short"/>
```

Comment:
Could you say more about this change? This doesn't make sense to me.

Reply:
In some examples for review articles, the previous code displays like "vol. ***", not "***", where "***" is a volume number. It is a reason for this change.

Reply:
That sounds like you just want to delete this line then? I'm not sure it would ever print something anyway?

Reply:
I will prepare the results for some examples, later. Please wait for my next reply. Thank you.

Reply:
I send you a PDF file, showing the inconsistent examples by the previous CSL, but the correct examples by the current CSL, which have just been displayed in Mendeley Deskt
[Mendeley_20211201.pdf](https://github.com/citation-style-language/styles/files/7630736/Mendeley_20211201.pdf)
o

Reply:
Two things:
1. My point above remains: if you just don't want the volume label, delete it
2. What item type are these in Mendeley? Because these really should map to article-journal in CSL, in which case all of this would not happen. My worry about this and the container-title change is that they now apply to all item types, which is clearly not right (e.g. you don't want the book title shortened for chapters and you *do* want the volume label for book volumes)

Reply:
Could you take a screenshot of the "Details" panel for the items that produced the citations in Mendeley Desktop and post it here?